src/login/AzureLoginHelper.ts

```diff
@@ -170,7 +165,7 @@ export class AzureAccountLoginHelper {
 		this.updateLoginStatus();
 	}
 
-	private async initialize(trigger: LoginTrigger, doLogin?: boolean, migrateToken?: boolean): Promise<void> {
+	private async initialize(trigger: LoginTrigger): Promise<void> {
```

Comment:
Removing the `initialize` call in `onDidChangeConfiguration` meant that these arguments were no longer needed along with a few of the `LoginTrigger` strings.